### ISSUE TITLE
Switching levels resets camera speed

### ISSUE THREAD

**Issue by jimbo1qaz** [magcius/noclip.website#129]
When I slide the camera speed slider all the way to the left, then switch stages, the camera speed slider remains at its new position, but the actual camera speed resets to the default value until I move the camera speed slider.

This is an issue on SSBB where the camera moves too quickly by default.

**Comment by magcius** [comment]
Yeah, camera controls are a bit finnicky. Planned rewrite to fix them.
